src/notification.js

```diff
@@ -0,0 +1,13 @@
+import { IncomingWebhook } from "@slack/webhook";
+
+const webhook = new IncomingWebhook('https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX');
+
+const sendNotification = async (invoice) => {
+    const {number, currency, amount, dueDate} = invoice;
+    await webhook.send({
```

Comment:
What if there is an exception thrown by the send function? Consider adding an exception handler.